vulnerabilities/tests/test_data/apache_kafka/to-advisory-apache_kafka-expected.json

```diff
@@ -0,0 +1,206 @@
+    ],
+    "references": [
+      {
+        "reference_id": "",
+        "url": "https://kafka.apache.org/cve-list",
+        "severities": []
+      },
+      {
+        "reference_id": "CVE-2021-38153",
+        "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-38153",
+        "severities": []
+      }
+    ],
+    "date_published": null,
+    "weaknesses": []
+  },
+  {
+    "aliases": [
+      "CVE-2019-12399"
+    ],
+    "summary": "When Connect workers in Apache Kafka 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.2.0, 2.2.1, or 2.3.0 are configured with one or more config providers, and a connector is created/updated on that Connect cluster to use an externalized secret variable in a substring of a connector configuration property value (the externalized secret variable is not the whole configuration property value), then any client can issue a request to the same Connect cluster to obtain the connector's task configurations and the response will contain the plaintext secret rather than the externalized secrets variable. Users should upgrade to 2.2.2 or higher, or 2.3.1 or higher where this vulnerability has been fixed.",
+    "affected_packages": [
+      {
+        "package": {
+          "type": "apache",
+          "namespace": null,
+          "name": "kafka",
+          "version": null,
+          "qualifiers": null,
+          "subpath": null
+        },
+        "affected_version_range": "vers:apache/2.0.0|2.0.1|2.1.0|2.1.1|2.2.0|2.2.1|!=2.2.2|2.3.0|<2.3.1",
+        "fixed_version": null
+      }
+    ],
+    "references": [
+      {
+        "reference_id": "",
+        "url": "https://kafka.apache.org/cve-list",
+        "severities": []
+      },
+      {
+        "reference_id": "CVE-2019-12399",
+        "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-12399",
+        "severities": []
+      }
+    ],
+    "date_published": null,
+    "weaknesses": []
+  },
+  {
+    "aliases": [
+      "CVE-2018-17196"
+    ],
+    "summary": "In Apache Kafka versions between 0.11.0.0 and 2.1.0, it is possible to manually craft a Produce request which bypasses transaction/idempotent ACL validation. Only authenticated clients with Write permission on the respective topics are able to exploit this vulnerability. Users should upgrade to 2.1.1 or later where this vulnerability has been fixed.",
+    "affected_packages": [
+      {
+        "package": {
+          "type": "apache",
+          "namespace": null,
+          "name": "kafka",
+          "version": null,
+          "qualifiers": null,
+          "subpath": null
+        },
+        "affected_version_range": "vers:apache/>=0.11.0.0|<2.1.1",
+        "fixed_version": null
+      }
+    ],
+    "references": [
+      {
+        "reference_id": "",
+        "url": "https://kafka.apache.org/cve-list",
+        "severities": []
+      },
+      {
+        "reference_id": "CVE-2018-17196",
+        "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-17196",
```

Comment:
Why do you use this URL? When I check the web page, I get things link https://nvd.nist.gov/vuln/detail/CVE-2022-34917

```HTML
<h2 id="CVE-2022-34917"><a href="https://nvd.nist.gov/vuln/detail/CVE-2022-34917">
CVE-2022-34917</a> Unauthenticated clients may cause OutOfMemoryError on brokers
</h2>

...
<h2 id="CVE-2022-23305"><a href="https://nvd.nist.gov/vuln/detail/CVE-2022-23305">
CVE-2022-23305</a> SQL injection Flaw in Apache Log4j logging library in versions 1.x
</h2>


```

Reply:
I've replaced mitre URLs with nist URLs.